README.md

```diff
@@ -2,160 +2,46 @@
-```
-
-Execute eos_state_validation playbook once change control has been approved and deployed to devices in CVP.
-Note: To run this playbook, ansible_host **must** be configured in your inventory for every EOS device. eAPI access **must** be configured and allowed in your networks.
+[Arista Networks](https://www.arista.com/) supports Ansible for managing devices running the Extensible Operating System (EOS) operating system natively through EOS API (eAPI) or [CloudVision Portal (CVP)](https://www.arista.com/en/products/eos/eos-cloudvision).
+This collection includes a set of ansible roles and modules to help kick-start your automation with Arista. The various roles and templates provided are designed to be customized and extended to your needs!
 
-```yml
-- hosts: DC1_FABRIC
+Full documentation for the collection:
 
-  tasks:
+- [Stable version](https://www.avd.sh/en/stable/)
+- [Development version](https://www.avd.sh/en/devel/)
 
-    - name: audit fabric state using EOS eAPI connection
-      import_role:
-         name: arista.avd.eos_validate_state
-```
+## Features
 
-**An example playbook to deploy VXLAN/EVPN Fabric via eAPI:**
+- **Flexibility with Open Data Models:** Extensible fabric-wide network models, simplifying configuration, delivering consistency, and reducing errors
+- **Simplification through Multi-Domain Automation:** A framework that can automate the data center, campus or wide area network, enabled by a consistent EOS software image and management platform
+- **Comprehensive Workflows:** Automating the full life cycle of network provisioning from config generation to pre and post-deployment validation, and self-documentation of the network
 
-![Figure 2: Example Playbook CloudVision Deployment](media/example-playbook-deploy-eapi.gif)
+## Reference designs
 
-```yml
-- hosts: DC1_FABRIC
+- [L3LS VXLAN-EVPN, L2LS, and MPLS (beta)](https://avd.sh/en/stable/roles/eos_designs/index.html)
 
-  tasks:
+## Project documentation
 
-    - name: generate intended variables
-      import_role:
-         name: arista.avd.eos_designs
+The documentation how to leverage ansible-avd collection is located here:
 
-    - name: generate device intended config and documentation
-      import_role:
-         name: arista.avd.eos_cli_config_gen
+- [Documentation](https://avd.sh/en/latest/)
+- [GitHub repository](https://github.com/aristanetworks/ansible-avd)
 
-    - name: deploy configuration via eAPI
-      import_role:
-         name: arista.avd.eos_config_deploy_eapi
+## Collection installation
 
-    - name: audit fabric state using EOS eAPI connection
-      import_role:
-         name: arista.avd.eos_validate_state
-```
+Ansible galaxy hosts all stable versions of this collection. Installation from ansible-galaxy is the most convenient approach for consuming `arista.avd` content. Please follow the collection installation [guide](https://avd.sh/en/stable/docs/installation/collection-installation.html).
```

Comment:
Could we use relative links here, so if ppl read this from github, they will stay on github, and if they read it from avd.sh, they stay there?

Reply:
same for other links

Reply:
Yes we can use relative links for the GitHub/Readme.md
We should keep http links on the collection/Readme.md since this is used in Ansible Galaxy: https://galaxy.ansible.com/arista/avd

Reply:
Corrected any links that weren't referencing external sources to be local.